backup/queries_relations.go

```diff
@@ -42,6 +42,25 @@ func relationAndSchemaFilterClause() string {
 	return filterRelationClause
 }
 
+func getExcludedRelationOidsList(connectionPool *dbconn.DBConn, quotedIncludeRelations []string) []string {
+	relList := utils.SliceToQuotedString(quotedIncludeRelations)
+
+	query := fmt.Sprintf(`
+	WITH root_oids AS (
```

Comment:
Does the 7.x branch should be supported? Seems, the `pg_partition_rule` seems was removed at 7.x, but other gpbackup code relies on it

Reply:
The gpbackup code has conditions only for 6x and only for 7x.

Reply:
Yes, and this script won't work on the 7.x branch due to pg_partition_rule. Sems new sync removes usage of `pg_partition_rule` at code base.

Reply:
I rewrote with using pg_inherits instead pg_partition_rule